blackbox/docs/configuration.txt

```diff
@@ -701,6 +701,23 @@ Collecting Stats
   expressive results but also in more occupied RAM. Setting it to
   ``0`` disables collecting job information.
 
+.. _stats.jobs_log_expiration:
+
+**stats.jobs_log_expiration**
+  | *Default:*  ``5s``
+  | *Runtime:*  ``yes``
+
+  This setting specifies the period which the jobs are kept in the ``sys.jobs_log``
+  table. It periodically clears entries that are finished and older than the
```

Comment:
Entries that are finished and older than the specified time interval are cleaned periodically.